src/main/java/nurseybook/logic/parser/EditTaskCommandParser.java

```diff
@@ -1,23 +1,18 @@
-package seedu.address.logic.parser;
+package nurseybook.logic.parser;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
```

Comment:
Similarly, here with `import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;` - would it be better?